client/package.json

```diff
@@ -13,7 +13,7 @@
         "url": "https://github.com/Microsoft/pyright"
     },
     "engines": {
-        "vscode": "^1.47.0"
+        "vscode": "^1.48.0"
```

Comment:
The engine and `@types/vscode` need to be kept in sync, that's why the other PR pipeline had failed.